nemo/lightning/pytorch/strategies/utils.py

```diff
@@ -21,6 +22,14 @@
 from nemo.utils.callbacks.dist_ckpt_io import AsyncFinalizableCheckpointIO
 
 
+@dataclass(kw_only=True)
+class SelectiveRestoreConfig:
```

Comment:
`RestoreConfig` seems like a better option, "Config" conveys the selective part by definition.